gapir/client/client.go

```diff
@@ -30,7 +30,7 @@ type tyLaunchArgsKey string
 
 // LaunchArgsKey is the bind device property key used to control the command
 // line arguments when launching GAPIR. The property must be of type []string.
-const LaunchArgsKey tyLaunchArgsKey = "<gapir-launch-args>"
+const LaunchArgsKey tyLaunchArgsKey = "gapir-launch-args"
```

Comment:
What's the purpose of this change? This constant is used as a property key, and there's no documentation / any obvious relation to the actual command line parameter name.

Reply:
Android doesn't seem to like <> characters in intent names. I played around with escaping them with some success, but it seemed like an unnecessary risk supporting exotic characters for what seemed like a cosmetic choice.

Reply:
Sure - my question was more: why is this being passed down to adb at all? It's an internal key for `gapir/client`. It appears that you've had to change the value because you've started using this a string to pass to adb, which is now doing more than it was intended to.

Reply:
I can create a new key for the intent if you'd prefer to keep the <> characters. I just thought it was simpler to use fewer identifiers around the place.

Reply:
There are other keys out there, none of them are reused for other string purposes, so I'd recommend just using the `"gapir-launch-args"` string literal at the site where you pass that at a parameter to the adb command, like all the other parameters.
My objection here is that there's nothing obvious or documented on this variable to say it is being used as a command line parameter string. Someone could decide they want to change the token value of `tyLaunchArgsKey` and unintentionally break the adb commands.

Reply:
This contant is mirrored in cmd/gapir/cc/main.cpp:596
```
const char* intent_flag = "gapir-launch-args";
```
This is fragile, as this string needs to exactly match the tyLaunchArgsKey.

I recommend to have a single declaration of the intent "keywork" as a different constant than tyLaunchArgsKey, and reuse that constant in both GAPIS and GAPIR, to make sure its actual value can be changed in a single place.

Reply:
> This is fragile, as this string needs to exactly match the tyLaunchArgsKey.

No. It should not have anything to do with `LaunchArgsKey`. :D

`gapir/client/session.go` has a whole bunch of command line parameter literal strings in it. Why is this one special?

If you create a whole bunch of named constants for command line parameters, okay - make them all constants, place them in `session.go`, but don't conflate it with device keys. They are completely unrelated concepts.

Reply:
To stress my point some more. `tyLaunchArgsKey` is a private type that is used purely as a unique map key with its single instance `LaunchArgsKey`. The only reason the underlying type is a string is so that it is easier to debug.
It could be an `int`, `struct`, `float64` - anything that produces a globally unique value-and-type key for the `bind.Registry.properties` map.
That's its soul purpose. Not to represent anything else.

Reply:
Changed to a string literal as @hevrard suggested at my desk

Reply:
Thanks Ben for the clarifications!